### PR TITLE
Feat: rework-dashboard-page

### PR DESCRIPTION
**Pull request by KevinLatino**
Feat: rework-dashboard-page
image: 
image

### CODE REVIEW

**Comment by djeck1432** [comment]
@KevinLatino Please, sync with main branch

**Comment by djeck1432** [comment]
1. Please add spaces appropriate to figma
Screenshot 2024-11-19 at 19 33 29
Screenshot 2024-11-19 at 19 33 40

**Comment by djeck1432** [comment]
Screenshot 2024-11-19 at 19 34 08
2. Please, add narrow (green and red) as it's in figma
Screenshot 2024-11-19 at 19 34 32

**Comment by djeck1432** [comment]
3.  add spaces as in figma
Screenshot 2024-11-19 at 19 34 43

**Comment by djeck1432** [comment]
Screenshot 2024-11-19 at 19 35 56
Color of text `Balance: ` is not the same as in figma, please fix it

File: frontend/src/pages/spotnet/dashboard/Dashboard.jsx [hunk 1]
```diff
@@ -1,183 +1,214 @@
...
+      currencyName: "Ethereum",
       currencyIcon: EthIcon,
     },
     {
-      title: 'Borrow',
+      title: "Borrow",
       icon: BorrowIcon,
-      balance: '0.00',
-      currencyName: 'USD Coin',
+      balance: "2.00",
```
Comment (djeck1432) [inline frontend/src/pages/spotnet/dashboard/Dashboard.jsx:35]: Fix this balance, it should be 0.0

**Comment by KevinLatino** [comment]
Image: 
image

**Review by djeck1432 — APPROVED** [review]
Looks good to me

**Pull request merged**